lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php

```diff
@@ -195,18 +195,41 @@ public function tablesExist($tableNames)
     }
 
     /**
-     * Returns a list of all tables in the current database.
+     * Returns true if all the given tables exist. This method ignores filters.
+     *
+     * @param string|string[] $tableNames
+     *
+     * @return bool
+     */
+    public function tablesExistInUnfilteredSchema($tableNames) : bool
```

Comment:
From the API name, it is unclear if all tables must exist, or only one suffices.

`allTablesExistInSchema` would solve that